services/image/src/routes/image.ts

```diff
@@ -0,0 +1,42 @@
+import { Hono } from 'hono'
+import { cors } from 'hono/cors'
+import { allowedOrigin } from '@kodadot/workers-utils'
+import { Env } from '../utils/constants'
+import { getImageByPath, uploadCFI } from '../utils/cloudflare-images'
+import { vValidator } from '@hono/valibot-validator'
+import Hash from 'ipfs-only-hash'
+
+import {
+  UploadImage,
+  uploadImageRequestSchema,
+} from '../schemas/uploadImageRequestSchema'
+
+const app = new Hono<{ Bindings: Env }>()
+
+app.use('/*', cors({ origin: allowedOrigin }))
+
+app.post('/upload', vValidator('form', uploadImageRequestSchema), async (c) => {
```

Comment:
nit: no need to create a new file for `uploadImageRequestSchema` for now. just put it here

Reply:
fixed